src/menu/menu.tsx

```diff
@@ -598,7 +609,40 @@ type MenuGroupProps = Omit<NativeProps<HTMLDivElement>, 'className'> & {
     /**
      * A label to be shown visually and also used to semantically label the group.
      */
-    label: string
+    label: NonNullable<React.ReactNode>
+
+    /**
+     * An optional info element to be shown to the right of the label.
+     *
+     * This is useful and often used to:
+     * - Provide a link to any documentation related to the menu items in the group
+     * - Show a keyboard shortcut that triggers the menu items in the group
+     *
+     * It is strongly recommended that this should be a icon-only element. It is also strongly
```

Comment:
💭 Does "icon-only element" include also the case of a keyboard shortcut? I would imagine that to be more like text than an icon

Reply:
Answering to myself: we do actually mean icon (and probably don't want plain text, as that would not be styled)

In fact, in the todoist-web sample PR, we do refer to the "icon representation" (React component) of the keyboard shortcut instead of the plain text for that shortcut. All clear now 👌